src/services/member/plugins/export-data/service.ts

```diff
@@ -24,26 +24,22 @@ import {
 } from './utils/anonymize.utils';
 import { RequestDataExportService } from './utils/export.utils';
 
+@singleton()
 export class ExportMemberDataService {
-  async requestDataExport({
-    actor,
-    repositories,
-    fileService,
-    mailer,
-  }: {
-    actor: Actor;
-    repositories: Repositories;
-    fileService: FileService;
-    mailer: MailerDecoration;
-  }) {
+  private requestDataExportService: RequestDataExportService;
```

Comment:
I think it can be readonly
```suggestion
  private readonly requestDataExportService: RequestDataExportService;
```